membership-common/src/main/scala/com/gu/zuora/ZuoraSoapService.scala

```diff
@@ -110,7 +110,7 @@ class ZuoraSoapService(soapClient: soap.ClientWithFeatureSupplier)(implicit ec:
       command.accountId,
       command.paymentGateway,
     ) // We need to set gateway correctly because it must match with the payment method we'll create below
-    createMethodResult <- soapClient.extendedAuthenticatedRequest[CreateResult](new XmlWriterAction(command)(createPaymentMethodWrites))
+    createMethodResult <- soapClient.authenticatedRequest[CreateResult](new XmlWriterAction(command)(createPaymentMethodWrites))
```

Comment:
this (createPaymentMethod) is the only call that used the "extended" (actually shorter) timeout